packages/mysql-community-8.0-mroonga/yum/oracle-linux-8/Dockerfile

```diff
@@ -0,0 +1,37 @@
+ARG FROM=oraclelinux:8
+FROM ${FROM}
+
+ARG DEBUG
+
+ENV \
+  SCL=gcc-toolset-11
+
+RUN \
+  quiet=$([ "${DEBUG}" = "yes" ] || echo "--quiet") && \
+  dnf -y module disable mysql && \
+  dnf update -y ${quiet} && \
+  dnf install -y ${quiet} \
+    'dnf-command(builddep)' \
+    'dnf-command(download)' \
+    https://packages.groonga.org/almalinux/8/groonga-release-latest.noarch.rpm \
+    https://repo.mysql.com/mysql-community-minimal-release-el8-1.noarch.rpm && \
+  dnf builddep --enablerepo=mysql80-server-minimal-source,ol8_codeready_builder -y ${quiet} mysql-community-minimal && \
+  dnf download --enablerepo=mysql80-server-minimal-source -y ${quiet} --source mysql-community-minimal && \
+  dnf install --enablerepo=ol8_codeready_builder -y ${quiet} \
+    ${SCL} \
+    ${SCL}-annobin-plugin-gcc \
+    ccache \
+    groonga-devel \
+    groonga-normalizer-mysql-devel \
+    intltool \
+    libcurl-devel \
+    libtool \
+    make \
+    pkgconfig \
+    rpm-build \
+    wget \
+    which && \
+  dnf install -y ${quiet} https://repo.mysql.com/mysql80-community-release-el8-3.noarch.rpm && \
```

Comment:
Why do we need to install this multiple times?

Reply:
Ah, https://repo.mysql.com/mysql-community-minimal-release-el8-1.noarch.rpm isn't the same package.